projects/admin/src/app/record/custom-editor/document-editor/document-editor.component.ts

```diff
@@ -118,4 +121,12 @@ export class DocumentEditorComponent {
       });
     }
   }
+
+  /**
+   * Can deactivate changed on editor
+   * @param activate - boolean
+   */
+  canDeactivateChanged(activate: boolean): void {
+    this.canDeactivate = activate;
+  }
```

Comment:
maybe this could be part of the `ng-core` abstract component ?

Reply:
An excellent proposal 👍🏻